src/foam/lib/json/FObjectArrayParser.java

```diff
@@ -15,7 +15,7 @@ public FObjectArrayParser() {
                    new Literal("["),
                    new Whitespace(),
                    new Repeat(
-                              new FObjectParser(),
+                              new ExprParser(),
```

Comment:
The `ExprParser` consists of an `Alt` with the two options being an `FObjectParser` and a custom parser for properties (`{ class: "__Property__". forClass_: "xyz" }`). This is needed to ensure properties are also parsed and handled correctly (the `__Property__` class isn't available from `foam.lookup()`, so the `FObjectParser` crashes if left as is).